helm/visualization/Chart.yaml

```diff
@@ -1,15 +1,16 @@
-apiVersion: v1
+apiVersion: v2
 description: A Helm chart for Kubernetes
```

Comment:
A helm chart for the service which is responsible for visualization of high dimensional data